proxies/live/apiproxy/targets/ers-target.xml

```
@@ -118,6 +160,15 @@
             <Step>
                 <Name>OauthV2.VerifyAccessToken</Name>
             </Step>
+            <!-- Must be placed after Authentication -->
+            <Step>
+                <Name>FlowCallout.ExtendedAttributes</Name>
+                <Condition>(accesstoken.auth_type == "user")</Condition>
+            </Step>
+            <Step>
+                <Name>FlowCallout.EUOAllowlistVerify</Name>
+                <Condition>(accesstoken.auth_type == "user")</Condition>
+            </Step>
             <Step>
                 <Name>RaiseFault.MissingAsid</Name>
```

CGI Feedback: Keep in mind the step order and the error being returned -- you only get one error, so priorities the error you want the user to see first

---

do we want the asid validation to come before this?

---

I think we can go both ways: if the partner ASID is missing, we could surface that error, but if possibly the verification logic is more specific, so we'd be going from that to the (possibly) more broader error or the partner ASID being missing in the request. The key point I think is that we'd terminate early, so a request with both issues won't progress until the first is resolved